activity.py

```diff
@@ -14,9 +22,47 @@ def __init__(self, handle):
         activity.Activity.__init__(self, handle)
 
         self.max_participants = 1
+        self.build_toolbar()
         self.game = conozco.Conozco(self)
         self.game.canvas = sugargame.canvas.PygameCanvas(self,
             main=self.game.principal,
             modules=[pygame.display, pygame.font])
         self.set_canvas(self.game.canvas)
         self.game.canvas.grab_focus()
+
+    def build_toolbar(self):
+        toolbar_box = ToolbarBox()
+
+        activity_button = ActivityButton(self)
+        toolbar_box.toolbar.insert(activity_button, 0)
+        activity_button.show()
+
+        description_item = DescriptionItem(self)
+        toolbar_box.toolbar.insert(description_item, -1)
+        description_item.show()
+
+        share_button = ShareButton(self)
+        toolbar_box.toolbar.insert(share_button, -1)
+        share_button.show()
```

Comment:
This button is not required.
[ShareButton](https://github.com/sugarlabs/sugar-toolkit/blob/16e7854c139cd0c9c720a27754ad14ff3ea7a634/src/sugar/activity/widgets.py#L145) definition disables the button because max_participants = 1 in this activity

Reply:
Thanks for noting that; I might have copied it from elsewhere;

Reply:
Have addressed this in https://github.com/sugarlabs/iknowSriLanka/pull/5/commits/3d2a7bb1bd3de7b97ead571d9e30ace33e756dec